third_party/.gitignore

```diff
@@ -10,6 +10,8 @@
 *
 !.gitignore
 !expat/
+!glfw/
```

Comment:
Let me know if you want me to update the comment in this file (see collapsed context); even pulling things via DEPS, we still need support files such as BUILD.gn that shouldn't be ignored, which means this file does need to be edited in that scenario.

Reply:
If you want to keep the sources in one spot but not have the BUILD.gn files mixed in with these sources, you can put them in the secondary buildroot. So you can add glfw/BUILD.gn and others to [//build/secondary/third_party](https://github.com/flutter/buildroot/tree/master/build/secondary/third_party) and not have to do any of this ignore wrangling.

Reply:
Done.